src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -578,4 +582,21 @@ public static void fillNormal(int[] faceData, EnumFacing facing)
         if(part.isPresent()) return Optional.absent();
         return Optional.of(new TRSRTransformation(matrix));
     }
+
+    public static ItemStackEffectPassHandler getStackGlintEffectPassHandler(ItemStack stack)
+    {
+        EffectOverlayEvent.ItemStackEffectOverlayEvent event = new EffectOverlayEvent.ItemStackEffectOverlayEvent(stack);
+        MinecraftForge.EVENT_BUS.post(event);
```

Comment:
Firing an event for each item each frame is probably not a good idea - enchanting glint puts the noticeable dent in the FPS already.